pages/tool_assembly/xnat_pic_tool_assembly.md

```diff
@@ -0,0 +1,52 @@
+---
+title: XNAT-PIC - Extending XNAT to Preclinical Imaging Centers
+search_exclude: true
+contributors: [Sara Zullino, Alessandro Paglialonga, Walter Dastrù, Dario Longo, Silvio Aime.]
+tags: [Preclinical  Imaging,  XNAT, Magnetic  Resonance  Imaging,  Image  Processing,  OpenScience, Database]
```

Comment:
Sorry these tags can only refer to Problem page tags that exist. The supported problem page tags are:
`compliance, data analysis, metadata, data quality, transfer, licensing, data organisation, sensitive, data publication, existing data, data protection, storage, identifiers, DMP`